### PR TITLE
bloodyAD: initial commit

### LINKED ISSUE #3329: new tool: bloodyAD
- **Name**: BloodyAD
- **Source URL**: https://github.com/CravateRouge/bloodyAD
- **Description**: BloodyAD is an Active Directory Privilege Escalation Framework

### PR DESCRIPTION
**Pull request by D4imyo**
fixes #3329

### CODE REVIEW

File: packages/bloodyAD/PKGBUILD [hunk 1]
```diff
@@ -0,0 +1,55 @@
+# This file is part of BlackArch Linux ( https://www.blackarch.org/ ).
+# See COPYING for license details.
+
+pkgname=bloodyad
```
Comment (noraj) [inline packages/bloodyAD/PKGBUILD:4]: LGTM but need to test it

File: packages/python-neo4j/PKGBUILD [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+pkgname=python-neo4j
+_pkgname=neo4j
+pkgver=4.4.5
+pkgrel=1
+pkgdesc='A Neo4j driver for Python.'
+arch=('any')
+url='https://pypi.org/project/neo4j/'
+license=('Apache2')
+depends=('python' 'python-pytz')
```
Comment (noraj) [inline packages/python-neo4j/PKGBUILD:12]: shouldn't it requires neo4j then? if it's a python wrapper for neo4j 

package: [neo4j-community](https://github.com/BlackArch/blackarch/blob/master/packages/neo4j-community/PKGBUILD)

Comment (noptrix) [reply packages/python-neo4j/PKGBUILD]: also, isn't it the `python-neo4j-driver` we already offer? i didn't check further. please check if it's the same package.

Comment (D4imyo) [reply packages/python-neo4j/PKGBUILD]: yeah turns out that they are the same package. thanks for pointing that out @noptrix

**Review by noraj — APPROVED** [review]
it's ok

**Comment by noptrix** [comment]
thanks guys

**Pull request merged**